metrotranscom/doorway
# fix: make drawer accessible in mobile landscape

#1012

## Description

Out of bash, we need on mobile devices in landscape mode looking at the map to be able to access the listings list. On production right now, you can see a little sliver, and on staging, you can't see it at all. This PR makes it so the list is accessible, but doesn't improve upon the experience.

This PR:
* Fixes the slight scrolling behavior in the old and the new map
* Creates more space on landscape by combining the filter bar and results bar in the old map
* Slightly reduces spacing around the logo on mobile

## How Can This Be Tested/Reviewed?

Using your mobile device, visit the listings map on landscape, and ensure the list is accessible. You can compare the experience to production. There are instructions [here](https://www.notion.so/exygy/How-to-visit-localhost-on-a-mobile-device-4857ad5b56db4c4db20632bf097adb71?pvs=4) for visiting your local stack in a mobile device. You can also toggle landscape mode on and off in the browser's mobile emulator.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
